### DISCUSSION TITLE
Modify Modal overlay and other component styles.

### DISCUSSION THREAD

**Discussion by ivanjeremic** [adobe/react-spectrum#5369] [Q&A] [answered]
I know that react-spectrum is meant to be used for adobe products but I would like to use it for my app which is not for any adobe product, is it possible to change parts of components like the Modal Overlay for example I would like to have some backdrop but for now I was not able to make it work. What is the best way to change it or is spectrum just not the right choice for non adobe products?

**Comment by reidbarber** [comment, marked as answer]
React Spectrum doesn't support too much in the way of style customization. You can read [this page](https://react-spectrum.adobe.com/react-spectrum/styling.html) for more information about why, and see what kind of styling is supported.

React Aria is going to be a better choice if you want to use custom styles. Check out the React Aria Components [Modal](https://react-spectrum.adobe.com/react-aria/Modal.html) docs page, and also the [Examples](https://react-spectrum.adobe.com/react-aria/examples/index.html) page for a few more examples. Also see the React Aria [Styling](https://react-spectrum.adobe.com/react-aria/styling.html) page for the different ways React Aria Components can be styled.
